docs/source/en/perf_infer_gpu_one.md

```diff
@@ -33,6 +33,7 @@ We natively support Flash Attention 2 for the following models:
 
 - Llama
 - Falcon
+- GPTBigCode (Starcoder)
```

Comment:
Same comment as for Llama, let's add a section in the readme of starcoder about expected gains!